src/domain/services/auth/revalidate-token.spec.ts

```
@@ -0,0 +1,109 @@
+      userAgent: session.userAgent ?? '',
+    })
+
+    expect(result).toBeDefined()
+    expect(result.status).toBe('error')
+    expect(result.code).toBe(401)
+    if (result.status !== 'error') throw new Error('Token was not revoked')
+
+    expect(result.message).toBe('Session revoked')
+  })
+
+  it.only('should not be able to revalidate with an invalid session ID', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)